app/models/planning_application.rb

```diff
@@ -215,6 +215,14 @@ def full_address
     "#{address_1}, #{town}, #{postcode}"
   end
 
+  def secure_change_url(local_authority, application_id, secure_token)
+    if ENV["RAILS_ENV"] == "production" || ENV["RAILS_ENV"] == "preview"
+      "https://#{local_authority}.#{ENV['applicants_app_host']}/change_requests?planning_application_id=#{application_id}&change_access_id=#{secure_token}"
```

Comment:
could this be `local_authority_name`? I think of it as the local_authority model object otherwise.

Reply:
In fact, seeing as we're inside the planning_application model, we can just call `local_authority.subdomain`, and we don't need to pass the name through at all?